libs/sdk-backend-tiger/src/convertors/toBackend/afm/TotalsConverter.ts

```diff
@@ -19,13 +26,16 @@ import { dimensionLocalIdentifier } from "./DimensionsConverter";
  * but such totals are impossible to define in the execution definition.
  */
 export function convertTotals(def: IExecutionDefinition): Total[] {
+    const { buckets } = def;
```

Comment:
nitpick: when this was introduced, consider enriching to `const { buckets, dimensions } = def`; and use dimensions below without accessing def